### PR TITLE
consolidated events controller and views

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -11,13 +11,12 @@ def index
...
     @group = Group.at_path(params[:group_path])
     @event = @group.events.new(event_params)
-    date = params[:date]
     date = DateTime.new(
-      date[:year].to_i,
-      date[:month].to_i,
-      date[:day].to_i,
-      date[:hour].to_i,
-      date[:minute].to_i
+      event_params["date(1i)"].to_i,
```
Comment (jshawl) [inline app/controllers/events_controller.rb]: can we use a date field or something? this feels code-smelly

**Comment by jshawl** [comment]
:+1: pending comment

**Comment by RobertAKARobin** [comment]
Sure! But I'm not sure how... Is there an alternative to `<select>` date menus?

**Comment by jshawl** [comment]
what about

```html
<input type="hidden" value="unix time stamp">
```

then in params parse to datetime, from beginning of day

**Comment by RobertAKARobin** [comment]
I dunno... That'll be just as much business logic as with this `datetime`, only in Javascript. I'd rather trust Rails / user what's ready.

**Pull request merged**